includes/admin/post-types/class.llms.meta.boxes.php

```diff
@@ -188,6 +189,68 @@ public function hide_meta_boxes() {
 
 	}
 
+	/**
+	 * Sync awarded certificates action.
+	 *
+	 * @since [version]
+	 *
+	 * @param WP_Post $post The WP_Post instance of the certificate template.
+	 * @return void
+	 */
+	public function sync_awarded_certificates_action( $post ) {
+
+		if ( 'llms_certificate' !== $post->post_type ) {
+			return;
+		}
+
+		$awarded_certificates_number = ( new LLMS_Awards_Query(
+			array(
+				'fields'    => 'ids',
+				'templates' => $post->ID,
+				'per_page'  => 1,
+				'status'    => array(
+					'publish',
+					'future',
+				),
+				'type'      => 'certificate',
+			)
+		) )->get_found_results();
+
+		if ( ! $awarded_certificates_number ) {
+			return;
+		}
+
+		$base_url = remove_query_arg( 'action' ); // Current url without 'action' arg.
+		$sync_url = add_query_arg(
+			'action',
+			'sync_awarded_certificates',
+			wp_nonce_url( $base_url, 'llms-cert-sync-actions', '_llms_cert_sync_actions_nonce' )
+		);
+
+		// Translators: %1$d = Number of awarded certificates.
+		$sync_alert = sprintf(
+			__( 'This action will replace the current title, content, and the background of %1$d awarded certificates with the ones of this template.\nAre you sure you want to proceed?', 'lifterlms' ),
+			$awarded_certificates_number
+		);
+		$on_click   = "return confirm('${sync_alert}')";
+
+		printf(
+			'<div class="llms-mb-section misc-pub-section sync-action"> %1$s</div>',
+			// Translators: %1$s = Opening anchor tag, %2$d = Closing anchor tag, %3$d = Number of awarded certificates.
+			sprintf(
+				__( '%1$sSync%2$s %3$d awarded certificates', 'lifterlms' ),
+				sprintf(
+					'<a href="%1$s" onclick="%2$s">',
+					$sync_url,
+					$on_click
+				),
+				'</a>',
+				$awarded_certificates_number
+			)
+		);
```

Comment:
This string desires some `_n()` usage, eg "1 awarded certificates" on my site currently.

Also... we should load the post name strings from the post type as to avoid multiple translations unnecessarily.

Eg: `get_post_type_object()->labels` etc...

Reply:
So I won't use _n since we're loading the post type labels...
Let's try this way.